node-server/src/services/synchronizer.js

```diff
@@ -45,16 +43,7 @@ class Synchronizer {
     let messageCount = 0
     Object.keys(messageDiff).forEach(key => {
       messageDiff[key].forEach(message => {
-        const dbMessage = {
-          nodeId: message.node_id,
```

Comment:
Removing this translation breaks the system, since there is the new check for `nodeId` being present.